examples/visualization/sensor_noise_level.py

```diff
@@ -15,13 +15,13 @@
 
 # %%
 
-import os.path as op
 import mne
 
 data_path = mne.datasets.sample.data_path()
 
-raw_erm = mne.io.read_raw_fif(op.join(data_path, 'MEG', 'sample',
-                                      'ernoise_raw.fif'), preload=True)
+raw_erm = mne.io.read_raw_fif(
+    data_path / 'MEG' / 'sample' / 'ernoise_raw.fif', preload=True
```

Comment:
I would suggest to put the preload on a new line

Reply:
I formatted it as `black` would, and it does not break arguments into multiple lines if they fit on a single line.